erpnext/accounts/doctype/payment_request/test_payment_request.py

```
@@ -851,3 +852,1027 @@ def test_payment_request_on_unreconcile(self):
+			return_value=True,
+		):
+			with patch(
+				"erpnext.accounts.doctype.payment_request.payment_request.PaymentRequest._process_v2_gateway"
+			):
+				with patch(
+					"erpnext.accounts.doctype.payment_request.payment_request.PaymentRequest.send_email"
+				) as mock_send_email:
+					with patch(
+						"erpnext.accounts.doctype.payment_request.payment_request.PaymentRequest.make_communication_entry"
+					) as mock_make_comm:
+						pr = make_payment_request(
```

Ruff (0.14.13)

Local variable `pr` is assigned to but never used

Remove assignment to unused variable `pr`

(F841)